src/main/java/eu/dissco/orchestration/backend/service/MappingService.java

```diff
@@ -36,4 +41,30 @@ public MappingRecord updateMapping(String id, Mapping mapping, String userId) {
       return null;
     }
   }
+
+  public JsonApiWrapper getMappings(int pageNum, int pageSize, String path){
+    var mappingRecords = repository.getMappings(pageNum, pageSize+1);
+    return wrapResponse(mappingRecords, pageNum, pageSize, path);
+  }
+
+  private JsonApiWrapper wrapResponse(List<MappingRecord> mappingRecords, int pageNum,
+      int pageSize, String path) {
+    boolean hasNext;
```

Comment:
Could this piece of logic move to an abstract class or maybe in the utility class? Assume this same logic is used in the SourceSystemService